clients/admin-ui/src/features/config-wizard/AddSystemForm.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionItem,
+  AccordionPanel,
+  chakra,
+  FormControl,
+  Heading,
+  HStack,
+  IconButton,
+  Stack,
+  Text,
+  Tooltip,
+} from "@fidesui/react";
+import type { NextPage } from "next";
+import React from "react";
+
+import {
+  AWSLogoIcon,
+  ManualSetupIcon,
+  OktaLogoIcon,
+  QuestionIcon,
+} from "~/features/common/Icon";
+
+const AddSystemForm: NextPage<{
+  handleChangeStep: Function;
+}> = ({ handleChangeStep }) => (
+  <chakra.form
+  // onSubmit={handleSubmit}
+  >
+    <Stack ml="50px" spacing="24px" w="80%">
+      <Heading as="h3" size="lg">
+        Add a system
+      </Heading>
+      <Accordion allowToggle border="transparent">
+        <AccordionItem>
+          {({ isExpanded }) => (
+            <>
+              <h2>
+                The building blocks of your data map are the list of systems
+                that exist in your organization. Think of systems as anything
+                that might store or process data in your organization, from a
+                web application, to a database or data warehouse.
+                <AccordionButton
+                  display="inline"
+                  padding="0px"
+                  ml="5px !important"
+                  width="auto"
+                >
+                  {isExpanded ? (
+                    <Text display="inline" color="complimentary.500">
+                      (show less)
+                    </Text>
+                  ) : (
+                    <Text display="inline" color="complimentary.500">
+                      (show more)
+                    </Text>
+                  )}
```

Comment:
nit: might be more concise as

```suggestion
                  <Text display="inline" color="complimentary.500">
                    { isExpanded ? (show less) : (show more) }
                  </Text>
```